### PR TITLE
Adding ExtensionsTimeBudget to new Api Version

### PR DESCRIPTION
**Pull request by kamusta-msft**
Squashed commits:

[f0151eb78] updated with extensionsTimeBudget

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAScaleSetWithExtensionsTimeBudget.json
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAScaleSetWithExtensionsTimeBudget.json:1]: You need add the link of this example file to compute.json file similar to below:

"$ref": "./examples/CreateAnAvailabilitySet.json"

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAVmInWithExtensionsTimeBudget.json
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAVmInWithExtensionsTimeBudget.json:1]: You need add the link of this example file to compute.json file similar to below:
"$ref": "./examples/CreateAnAvailabilitySet

**Review by hyonholee — CHANGES_REQUESTED** [review]
Please add the links of new example files to compute.json.

**Comment by ArcturusZhang** [comment]
Since this PR is not targeting master branch but a dev branch of compute (I assume), I will then merge this PR as long as @hyonholee approves.

**Comment by ArcturusZhang** [comment]
Hi @kamusta-msft @hyonholee any updates?

**Comment by hyonholee** [comment]
Please resolve the conflict

**Comment by hyonholee** [comment]
@ravbhatnagar Could you review this PR?

**Review by pilor — APPROVED** [review]
:shipit:

**Comment by ArcturusZhang** [comment]
Hi @hyonholee could you please approve? I would merge this once you approved this, thanks!

**Review by ArcturusZhang — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAVmInWithExtensionsTimeBudget.json
Comment (Sandido) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAVmInWithExtensionsTimeBudget.json:1]: Since you're adding this file, the name CreateAVmInWithExtensionsTimeBudget seems odd. Unless this is a standard naming schema, remove 'In'.

**Review by Sandido — CHANGES_REQUESTED** [review]
See my comment about removing the 'In' string from the file name.

**Review by hyonholee — APPROVED** [review]

**Comment by Sandido** [comment]
@kamusta-msft , now that this Swagger PR has been merged, you can create a PR in the relevant SDK repository with the autogenerated SDK and at least 1 test added. Please reach out to me if you're not sure how to proceed.

**Pull request merged**
